packages/components/alert/src/alert.ts

```diff
@@ -61,11 +61,7 @@ export const alertProps = buildProps({
 
 export type AlertProps = ExtractPropTypes<typeof alertProps>
 
-export const alertEmits = {
-  click: (event: Event) => event instanceof Event,
-  close: (event: Event) => event instanceof Event,
-}
-
+export const alertEmits = ['click', 'close']
```

Comment:
Mmmh so click or close could be a function, a string, a number ?

Reply:
just string
alertEmits is then imported into the vue file and used as an argument to defineEmits fucntion which expects an array of strings
`const emit = defineEmits(alertEmits)`